### PR TITLE
media-gfx/digikam: Add USE=opencv3, translations and doc

### PR DESCRIPTION
**Pull request by a17r**
Built fine.

### CODE REVIEW

File: eclass/kde5-functions.eclass [hunk 1]
```diff
@@ -300,6 +300,23 @@ kde_l10n2lingua() {
 	done
 }
 
+# @FUNCTION: prune_doc_linguas
+# @USAGE: <docdir>
+# @DESCRIPTION:
+# Removes all but wanted LINGUAS from specified doc dir.
+# No action if LINGUAS is unspecified.
+prune_doc_linguas() {
```
Comment (Arfrever) [inline eclass/kde5-functions.eclass]: Add `local lang` here.

**Comment by a17r** [comment]
see also: https://github.com/gentoo/gentoo/pull/2009

File: eclass/kde5.eclass [hunk 2]
```diff
@@ -534,17 +535,20 @@ EOF
 				fi
 			fi
 		done
+		unset lang
```
Comment (Arfrever) [inline eclass/kde5.eclass]: When `local` is used for a variable, then previous state of this variable (e.g. it being unset or its previous value if it was set) will be automatically restored after finishing of execution of given function.

File: eclass/kde5.eclass [hunk 3]
```diff
@@ -534,17 +535,20 @@ EOF
...
+	if [[ ${KDE_BUILD_TYPE} = release && ${CATEGORY} != kde-apps ]] ; then
+		if [[ ${KDE_HANDBOOK} != false && -d ${KDE_DOC_DIR} && -v LINGUAS ]] ; then
 			pushd ${KDE_DOC_DIR} > /dev/null || die
+			local lang
 			for lang in *; do
 				if ! has ${lang} ${LINGUAS} ; then
 					cmake_comment_add_subdirectory ${lang}
 				fi
 			done
+			unset lang
```
Comment (Arfrever) [inline eclass/kde5.eclass]: Same here.

**Comment by Arfrever** [comment]
Maybe this USE flag is not needed at all?. You can use:
`-DENABLE_OPENCV3=$(has_version ">=media-libs/opencv-3" && echo yes || echo no)`

And `media-libs/opencv:=` in dependencies.

**Comment by kensington** [comment]
Radical! Picked as 95cd09d47a77be8667dbf1d535a86ffdc151ffdf...774151db1c8bf23e3e0fbc65b418d60c48f775c5.

**Pull request closed without merging**